lib/galaxy/webapps/galaxy/api/jobs.py

```diff
@@ -343,10 +343,17 @@ def error(self, trans, id, **kwd):
         # Get job
         job = self.__get_job(trans, id)
         tool = trans.app.toolbox.get_tool(job.tool_id, tool_version=job.tool_version) or None
+        email = kwd.get('email')
+        if not email and not trans.anonnymous:
```

Comment:
`trans.anonymous` I think (extra `n` here)

Reply:
Oops, thanks, I've rebased the commit with the fix and I'm testing that route as well now.